frontend/src/Pages/StudentManagement.tsx

```diff
@@ -177,22 +176,21 @@ export default function StudentManagement() {
                         data-tip="Add Student"
                     >
                         <button
-                            className="btn btn-primary btn-sm"
+                            className="btn btn-primary btn-sm text-base-teal"
                             onClick={() => addUserModal.current?.showModal()}
                         >
-                            <UserPlusIcon className="h-4" />
+                            <PlusCircleIcon className="w-4 my-auto" />
+                            Add Student
                         </button>
                     </div>
                 </div>
-                <table className="table">
+                <table className="table-2">
                     <thead>
-                        <tr className="border-gray-600">
-                            <th className="flex">
-                                <span>Name</span>
-                            </th>
+                        <tr className="grid-cols-4 px-4">
+                            <th className="justify-self-start">Name</th>
                             <th>Username</th>
-                            <th>Last Updated</th>
-                            <th>Actions</th>
+                            <th>Last Active</th>
```

Comment:
Keep this as "Last Updated".. "Last Active" is not an accurate label column because the user is not updated with data when they are active!